<p>
  The following error prevented a file from pushing to the workspace bucket and launching an ingest pipeline run:
</p>
<h3>Error</h3>
<p><strong><%= @error.class.name %></strong> - <%= @error.message %></p>
<h3>Job Details</h3>
<ul>
  <li>Associated User: <%= @user.email %></li>
  <li>Study: <%= @study.accession %></li>
  <li>Workspace: <%= link_to @study.workspace_url, @study.workspace_url %></li>
  <li>Bucket:  <%= link_to @study.google_bucket_url, @study.google_bucket_url %></li>
  <li>Internal workspace: <%= link_to @study.workspace_url(:internal), @study.workspace_url(:internal) %></li>
  <li>Internal bucket:  <%= link_to @study.google_bucket_url(:internal), @study.google_bucket_url(:internal) %></li>
  <li>File: <%= @study_file.upload_file_name %></li>
  <li>File Type: <%= @study_file.file_type %></li>
  <li>Uploaded/synced on: <%= @study_file.created_at.to_s(:db) %></li>
  <li>File still local? <%= @study_file.is_local? %></li>
</ul>

<p>
  Once the error has been resolved, please relaunch the file from the Rails console with the following commands:
</p>

<pre><strong>
  study = Study.find_by(accession: '<%= @study.accession %>')
  study_file = study.study_files.detect {|file| file.upload_file_name == '<%= @study_file.upload_file_name %>'}
  user = User.find_by(email: '<%= @user.email %>')
  study_file.update(parse_status: 'unparsed')
  FileParseService.run_parse_job(study_file, study, user)
</strong></pre>
